StockCount/View/OptionViewController.swift

```diff
@@ -19,16 +19,15 @@ class OptionViewController : UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        let versionString = (Bundle.main.infoDictionary?["CFBundleVersion"] as? String) ?? ""
-        lblVersion.text = "version".localized + versionString + "\nCapture ver"
+        CaptureHelper.sharedInstance.dispatchQueue = DispatchQueue.main
+        let appVer = (Bundle.main.infoDictionary?["CFBundleVersion"] as? String) ?? ""
+        updateVersion(appVer, "fetching".localized)
         CaptureHelper.sharedInstance.getVersionWithCompletionHandler { (result, version) in
-            DispatchQueue.main.async {
-                let majorVersionString = String(version?.major ?? 0)
-                let middleVersionString = String(version?.middle ?? 0)
-                let minorVersionString = String(version?.minor ?? 0)
-                let captureVersion = majorVersionString + "." + middleVersionString + "." + minorVersionString
-                self.lblVersion.text = "version".localized + versionString + "\nCapture \(captureVersion)"
+            var captureVersion = "error".localized
```

Comment:
we should check if result is successful. If it is then version will be correct.

Reply:
I don't see the check for result success?

Reply:
updated code on last commit.

https://github.com/SocketMobile/stockcount-ios/pull/35/commits/8c59739b1b2668f2d58b0b2cce9f715e1aed9958